src/main/java/uk/gov/pay/directdebit/common/model/ErrorResponse.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.pay.directdebit.common.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include;
+
+@JsonInclude(Include.NON_NULL)
+public class ErrorResponse {
+    
+    @JsonProperty("error_identifier")
+    private final ErrorIdentifier identifier;
+    
+    @JsonProperty("message")
+    private final List<String> messages;
+    
+    @JsonProperty("reason")
```

Comment:
Maybe a `@Nullable` annotation (or `@NotNull` on the other fields) would be useful here

Reply:
I'll look at potentially making this optional if we share the model with connector public API via pay-java-commons. For now I'll just remove the reason field in case we decide against sharing it in the end.